project/new_project/export_project.js

```diff
@@ -71,7 +80,7 @@ function redirect_edit_page(username, branch) {
 	window.location.replace(`${home}/${path}`);
 }
 
-function redirect_new_page(username, branch, category) {
+function redirect_new_page(username, branch, category, filename) {
 	const home = `https://github.com/${username}/${username}.github.io/new/${branch}`
-	window.location.replace(`${home}/project/${category}/ongoing_projects`);
+	window.location.replace(`${home}/project/${category}/ongoing_projects?filename=${filename}`);
```

Comment:
`filename` is URL Parameter that GitHub will use to prepopulate the title of the newly generated file